common/lib/calc/setup.py

```diff
@@ -7,6 +7,6 @@
     install_requires=[
         "pyparsing==2.0.1",
         "numpy",
-        "scipy"
+        "scipy<0.18"
```

Comment:
Hi @nedbat and @blarghmatey, Thanks you both for this fix. We've just been facing this issue and I found the solution here. However, we followed @nedbat suggestion and pinned the versions with exact versions. Here's our local pull request that you might want to cherry-pick and use here: https://github.com/Edraak/edx-platform/pull/231